api/lib/source/cosmosV0-source.js

```diff
@@ -288,6 +290,66 @@ class CosmosV0API extends RESTDataSource {
     )
   }
 
+  async getRecentProposals() {
+    // while we don't have proposals in DB this is the only way
+    const proposals = await this.getAllProposals()
+    return proposals.sort(
+      (a, b) => new Date(b.creationTime) - new Date(a.creationTime)
+    )
+  }
+
+  async getTopVoters() {
+    // for now defaulting to pick the 5 largest voting powers
+    return _.take(
+      _.reverse(
+        _.sortBy(this.store.validators, [
+          (validator) => {
+            return validator.votingPower
+          }
+        ])
+      ),
+      5
+    ).map(({ operatorAddress }) => operatorAddress)
+  }
+
+  async getGovernanceOverview() {
+    const { bonded_tokens: totalBondedTokens } = await this.query(
+      '/staking/pool'
+    )
+    const [
+      communityPoolArray,
+      links,
+      recentProposals,
+      topVoters
+    ] = await Promise.all([
+      this.query('/distribution/community_pool'),
+      this.db.getNetworkLinks(this.network.id),
+      this.getRecentProposals(),
+      this.getTopVoters()
+    ])
+    const communityPool = communityPoolArray.find(
+      ({ denom }) => denom === this.network.coinLookup[0].chainDenom
+    ).amount
+    return {
+      totalStakedAssets: fixDecimalsAndRoundUpBigNumbers(
+        totalBondedTokens,
+        2,
+        this.network,
+        this.network.stakingDenom
+      ),
+      totalVoters: undefined,
+      treasurySize: fixDecimalsAndRoundUpBigNumbers(
+        communityPool,
+        2,
+        this.network,
+        this.network.stakingDenom
```

Comment:
why do you need to pass the stakingDenom when you already pass in the whole network?